composition/src/warnings/warnings.ts

```diff
@@ -130,3 +131,27 @@ export function requiresDefinedOnNonEntityFieldWarning(fieldCoords: string, subg
     },
   });
 }
+
+export function consumerInactiveThresholdValueShouldBePositiveWarning(subgraphName: string): Warning {
+  return new Warning({
+    message:
+      `The "consumerInactiveThreshold" argument of type "Int" should be positive and smaller than 2,147,483,647.\n` +
+      `The value has been set to ` +
+      DEFAULT_CONSUMER_INACTIVE_THRESHOLD +
+      `.`,
+    subgraph: {
+      name: subgraphName,
+    },
+  });
+}
+
+export function maxConsumerInactiveThresholdExceededWarning(subgraphName: string): Warning {
+  return new Warning({
+    message:
+      `The "consumerInactiveThreshold" argument of type "Int" should be positive and smaller than 2 147 483 647.\n` +
```

Comment:
The max int here needs formatting as above.

Also consider for this and the above: "should be a number greater than 0 and smaller than 2,147,483,648"`

Note that the max value is ...647, so it should be less than 648.

Reply:
`"consumerInactiveThresholdInvalidValueWarning"`

Reply:
The `This means the consumer will remain indefinitely active until its manual deletion.`` can be a conditional extra message